core/src/replay_stage.rs

```diff
@@ -1682,6 +1682,16 @@ impl ReplayStage {
         blockstore
             .set_dead_slot(slot)
             .expect("Failed to mark slot as dead in blockstore");
+
+        if let Some(stats) = blockstore.turbine_slots_stats_remove_slot(slot) {
+            let min_turbine_batch_count = stats.get_min_batch_count();
+            datapoint_warn!(
+                "replay-stage-mark_dead_completed_unrepaired_slot",
+                ("slot", slot, i64),
+                ("min_turbine_batch_count", min_turbine_batch_count, i64),
+            );
```

Comment:
changed to log in both cases

Reply:
Where is the logging for the other cases? i.e. if the slot is rooted and not dead.

Reply:
I would suggest lets not throw out some of the samples based on if there was repair or not.
Turbine works independently of the repair. Here we just want to measure turbine. Otherwise later if there is a change to repair logic (e.g. if nodes repair more aggressively), then the data here will change without any change to turbine.